### ISSUE TITLE
change test data set

### ISSUE THREAD

**Issue by MiataMike** [MiataMike/moneymonster#8]
dataset could be overfitting and inflating it's accuracy by testing against similar elements (chosen at random from total data set)

**Comment by MiataMike** [comment]
created kerasWtest.py, which does not scramble the data anymore. This should maintain the order in masterdata.csv (each stock ticker, with sequential slices)
Will close if there is a difference in testing accuracy

**Comment by MiataMike** [comment]
there doesn't seem to be a difference! leaving the code as is
